package.json

```diff
@@ -14,17 +14,21 @@
     "typecheck": "turbo run typecheck",
     "// round manager script": "====== packages/round-manager specific ======",
     "rm-build": "pnpm --filter round-manager run build",
-    "rm-test": "pnpm --filter round-manager run test",
+    "rm-test": "turbo run test --filter=round-manager",
     "rm-start": "pnpm --filter round-manager run start",
-    "rm-slither": "pnpm --filter round-manager run slither",
-    "rm-typecheck": "pnpm --filter round-manager exec  -- tsc --noEmit",
+    "rm-lint": "turbo run lint:ci --filter=round-manager",
+    "rm-typecheck": "turbo run typecheck --filter=round-manager",
     "// grant explorer script": "====== packages/grant-explorer specific ======",
     "re-build": "turbo run build --filter=grant-explorer",
     "re-test": "turbo run test --filter=grant-explorer",
     "re-start": "pnpm --filter grant-explorer run start",
     "re-typecheck": "turbo run typecheck --filter=grant-explorer",
+    "re-lint": "turbo run lint:ci --filter=grant-explorer",
     "// builder script": "====== packages/builder specific ======",
-    "b-start": "pnpm --filter builder run start"
+    "b-start": "pnpm --filter builder run start",
```

Comment:
it does, but since the whole point of turbo is to cache and skip tasks that shouldn't be run (like lint, build, test etc.) and we always want to run start, i would rather have pnpm manage that.